### PR TITLE
chore: fix breaking change PR release workflow

### PR DESCRIPTION
**Pull request by Torres-ssf**
- Close: https://github.com/FuelLabs/fuels-ts/issues/1929

### CODE REVIEW

**Comment by fuel-service-user** [comment]
✨ A PR has been created under the `rc-1928` tag on `fuels-wallet` repo.
https://github.com/FuelLabs/fuels-wallet/pull/1178

File: .github/workflows/breaking-change-pr-release.yaml [hunk 1]
```diff
@@ -4,10 +4,45 @@ on:
...
     branches:
       - "master"
+    types:
+      - opened
+      - edited
+      - synchronize
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
```
Comment (arboleya) [inline .github/workflows/breaking-change-pr-release.yaml:14]: Please, add a description to the PR.

**Review by arboleya — DISMISSED** [review]

File: .github/workflows/breaking-change-pr-release.yaml (hunk 1 above)
Comment (Torres-ssf) [reply .github/workflows/breaking-change-pr-release.yaml:15]: Sorry I've marked it as Ready for Review only to validate the desired workflow is being triggered, as we want to only happen when the PR is ready for review

Comment (Torres-ssf) [reply .github/workflows/breaking-change-pr-release.yaml:15]: I am still running some validations

**Review by arboleya — APPROVED** [review]

**Review by petertonysmith94 — APPROVED** [review]

**Pull request merged**